src/Polish.cpp

```diff
@@ -295,7 +318,15 @@ vector<int> ConsensusQualities(AbstractIntegrator& ai)
             if (m.Start() > i) continue;
             // TODO (lhepler): this is dumb, but untestable mutations,
             //   aka insertions at ends, cause all sorts of weird issues
-            const double score = ai.LL(m) - LL;
+            double score;
+            try {
+                score = ai.LL(m) - LL;
+            } catch (const Exception::InvalidEvaluatorException& e) {
+                // If an Evaluator exception occured, report and skip! 
+                // We need to handle this!
+                std::cerr << "In Polish::ConsensusQualities(ai):" << e.what() << std::endl;
```

Comment:
I think at the moment we don't have an official logging solution (or dependency injection for such) for unanimity/CC2. Until we can figure out how to do that, I'm not sure this is precisely what we should do...